MongoDB/containers/0.1.0/mongodb-config/connect.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
-DIG=/opt/rancher/bin/dig
+
+set -x
+
+GIDDYUP=/opt/rancher/bin/giddyup
 
 function cluster_init {
 	sleep 10
-	MYIP=$(ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1 |  sed -n 2p)
-	$DIG A $MONGO_SERVICE_NAME +short > ips.tmp
+	MYIP=$($GIDDYUP ip myip)
+	$GIDDYUP ip stringify --delimiter " " > ips.tmp
 	mongo --eval "printjson(rs.initiate())"
 	for member in $(cat ips.tmp); do
```

Comment:
something more like this, to avoid writing/reading a file
```
for member in $(giddyup ip stringify --delimiter " "); do
  if [ "$member" != "$MYIP" ]; then
   ...
  fi
done
```